src/libraries/System.Net.Quic/src/System/Net/Quic/Internal/MsQuicHelpers.cs

```diff
@@ -35,23 +35,12 @@ internal static bool TryParse(this EndPoint endPoint, out string? host, out IPAd
         return false;
     }
 
-    internal static unsafe IPEndPoint ToIPEndPoint(this ref QuicAddr quicAddress, AddressFamily? addressFamilyOverride = null)
-    {
-        // MsQuic always uses storage size as if IPv6 was used
-        Span<byte> addressBytes = new Span<byte>((byte*)Unsafe.AsPointer(ref quicAddress), Internals.SocketAddress.IPv6AddressSize);
-        return new Internals.SocketAddress(addressFamilyOverride ?? SocketAddressPal.GetAddressFamily(addressBytes), addressBytes).GetIPEndPoint();
-    }
-
     internal static unsafe QuicAddr ToQuicAddr(this IPEndPoint iPEndPoint)
```

Comment:
We should get rid off of both these helpers and replace them everywhere. Or keep them both in the simplified version and all they're usages.